chef/cookbooks/provisioner/templates/default/autoyast-upgrade.xml.erb

```diff
@@ -12,6 +12,7 @@
       <final_reboot config:type="boolean">false</final_reboot>
       <halt config:type="boolean">false</halt>
       <second_stage config:type="boolean">true</second_stage>
+      <ntp_sync_time_before_installation><%= @ntp_servers.first %></ntp_sync_time_before_installation>
```

Comment:
From [AutoYaST](https://www.suse.com/documentation/sles-12/singlehtml/book_autoyast/book_autoyast.html):

> Please keep in mind that you need a reachable NTP server and network connections while running the installation.

which sounds like the installation will fail in case of a missing NTP server.